src/Params.ml

```diff
@@ -68,9 +68,61 @@ let all ~first ~indent ~parens_here =
   ; break_after_opening_paren= fmt "@ "
   ; box_rhs= hovbox 0 }
 
-let get_cases c =
-  match c.Conf.break_cases with
+let get_cases (c : Conf.t) =
+  match c.break_cases with
   | `Fit -> fit
   | `Nested -> nested
   | `Toplevel -> toplevel
   | `All -> all
+
+type record_type =
+  { docked_before: Fmt.t
+  ; break_before: Fmt.t
+  ; box_record: Fmt.t -> Fmt.t
+  ; sep_before: Fmt.t
+  ; sep_after: Fmt.t
+  ; break_after: Fmt.t
+  ; docked_after: Fmt.t }
+
+let before (c : Conf.t) ~wrap_record =
```

Comment:
I think you should move these functions either in a module `Record_type` or, why not, inline them in `get_record_type`.

Reply:
I inlined the functions, using a module would be contrary to what was decided for `get_cases`.

Reply:
Thanks, it's more readable. Could you do the same for `get_cases` ?

Reply:
Done.